src/etc/inc/load_balancer_maintable.inc

```diff
@@ -155,11 +155,11 @@ class MainTable {
 				break;
 			}
 			case "del": {
-				echo "<a href=\"{$this->my_uri}?act=del&amp;id={$row}\" onclick=\"return confirm('Do you really want to delete this entry?')\" class=\"btn btn-default btn-xs\"><span class=\"glyphicon glyphicon-remove\"></span></a>";
+				echo "<a href=\"{$this->my_uri}?act=del&amp;id={$row}\" onclick=\"return confirm('".gettext('Do you really want to delete this entry?')."')\" class=\"btn btn-default btn-xs\"><span class=\"glyphicon glyphicon-remove\"></span></a>";
 				break;
 			}
 			case "dup": {
-				echo "<a href=\"{$this->edit_uri}?act=dup&amp;id={$row}\" class=\"btn btn-default btn-xs\" title=\"Duplicate entry\" alt=\"duplicate\" /><span class=\"glyphicon glyphicon-plus\"></span></a>";
+				echo "<a href=\"{$this->edit_uri}?act=dup&amp;id={$row}\" class=\"btn btn-default btn-xs\" title=\"".gettext('Duplicate entry')."\" alt=\"".gettext('duplicate')."\" /><span class=\"glyphicon glyphicon-plus\"></span></a>";
```

Comment:
might be better to remove alt= completely? Doesn't add much value and is not consistent with the rest of this switch statement.